.github/workflows/stale.yml

```diff
@@ -43,3 +43,4 @@ jobs:
         days-before-pr-stale: '10'
         days-before-pr-close: '20'
         days-before-issue-stale: '60'
+        days-before-issue-close: '67'
```

Comment:
Can we also edit `stale-issue-message` above to mention the close window? 
Something to the effect of "this issue will be closed in 7 days"?

Reply:
Applied. Thanks.